ui/pages/bridge/utils/validators.ts

```diff
@@ -53,6 +54,15 @@ export const FEATURE_FLAG_VALIDATORS = [
   },
 ];
 
+export const TOKEN_AGGREGATOR_VALIDATORS = [
+  {
+    property: 'aggregators',
+    type: 'object',
+    validator: (v: unknown): v is number[] =>
+      isValidObject(v) && Object.values(v).every(isValidString),
```

Comment:
Is this meant to be `isValidNumber`? Or is the return type wrong here? It's saying that it's validating these to be numbers, but it's not.

Reply:
Oh the return type is wrong either way. `v` is an object, not an array of numbers.